src/components/v5/shared/RolesTooltip/RolesTooltip.tsx

```diff
@@ -0,0 +1,50 @@
+import { ColonyRole } from '@colony/colony-js';
+import React, { FC } from 'react';
+
+import { USER_ROLE, USER_ROLES } from '~constants/permissions';
+import Tooltip from '~shared/Extensions/Tooltip';
+import { formatText } from '~utils/intl';
+import PermissionsBadge from '~v5/common/Pills/PermissionsBadge';
+import Link from '~v5/shared/Link';
+
+import { RolesTooltipProps } from './types';
+
+const RolesTooltip: FC<RolesTooltipProps> = ({
+  role,
+  roleDescription,
+  tooltipProps,
+  learnMoreProps: { link, label } = {},
+}) => {
+  return (
+    <Tooltip
+      tooltipContent={
+        <>
+          {roleDescription ??
```

Comment:
Is `??` really needed here? I think that `||` would be enough.

Reply:
Changed